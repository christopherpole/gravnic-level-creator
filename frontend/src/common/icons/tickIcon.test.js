import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import TickIcon from './tickIcon';

configure({ adapter: new Adapter() });

describe('<TickIcon />', () => {
  it('Renders without exploding', () => {
    const tickIcon = shallow(<TickIcon />);

    expect(tickIcon).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const tickIcon = shallow(<TickIcon />);

    expect(toJson(tickIcon)).toMatchSnapshot();
  });
});
