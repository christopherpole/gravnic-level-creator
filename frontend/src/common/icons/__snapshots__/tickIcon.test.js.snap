// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TickIcon /> Matches the current snapshot 1`] = `
<svg
  height="15px"
  viewBox="0 0 24 24"
  width="15px"
>
  <path
    d="M20.285 2l-11.285 11.567-5.286-5.011-3.714 3.716 9 8.728 15-15.285z"
    fill="green"
  />
</svg>
`;
